src/__experimental__/components/grouped-character/grouped-character.component.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Textbox from '../textbox';
+import { generateGroups, toSum, toIndexSteps } from './grouped-character.utils';
+import Events from '../../../utils/helpers/events/events';
+
+const GroupedCharacter = (props) => {
+  const [pressedKey, updatePressedKey] = useState(null);
+
+  const { separator, groups, value } = props;
+  const stepIndices = groups.reduce(toIndexSteps, []);
+
+  const handleChange = (e) => {
+    const i = e.target;
+    const { selectionEnd } = e.target;
+    let modifier = 0;
+
+    if (stepIndices.includes(selectionEnd - 1)) modifier = 1;
+    if (stepIndices.includes(selectionEnd - 1) && Events.isBackspaceKey({ which: pressedKey })) modifier = -1;
+
+    props.onChange({ target: { value: e.target.value.split(separator).join('') } });
+    setTimeout(() => i.setSelectionRange(selectionEnd + modifier, selectionEnd + modifier));
+  };
+
+  return (
+    <Textbox
```

Comment:
Should we be using the InputDecoratorBridge here?

Reply:
My spec was to use the new TextBox, why would you suggest using the bridge?